.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+C:/Users/Eric/node_modules/
```

Comment:
- [OPTIONAL] Please you have already specified `node_modules/` so it is better you remove this absolute path `C:/Users/Eric/node_modules/`